src/hb-ot-name-table.hh

```diff
@@ -97,12 +142,39 @@ struct NameRecord
     return UNSUPPORTED;
   }
 
-  NameRecord* copy (hb_serialize_context_t *c, const void *base) const
+  NameRecord* copy (hb_serialize_context_t *c, const void *base,
+                    const hb_hashmap_t<unsigned, const char*> *name_table_overrides) const
   {
     TRACE_SERIALIZE (this);
+    auto snap = c->snapshot ();
     auto *out = c->embed (this);
     if (unlikely (!out)) return_trace (nullptr);
-    out->offset.serialize_copy (c, offset, base, 0, hb_serialize_context_t::Tail, length);
+    if (name_table_overrides->has (nameID)) {
```

Comment:
This branch should also set platform, etc, to make sure they are what we expect. But then the sort order might go wrong :(.. Humm... Maybe is okay to assume they are correct. Ouch.

Also, currently, names corresponding to the nameID, in ALL languages, will be overriden.

In general this API is very brittle and I don't like it. I'm happy to leave it in experimental for now though.